buildfix.sh

```diff
@@ -40,39 +65,46 @@ if which clang-format &>/dev/null; then
   echo "Formatting .proto files..."
   git ls-files --exclude-standard | grep '\.proto$' | xargs -d '\n' --no-run-if-empty clang-format -i --style=Google
 else
-  echo -e "${c_yellow}WARNING: Missing clang-format; will not format proto files.${c_reset}"
+  echo -e "${c_yellow}WARNING: Missing clang-format tool; will not format proto files.${c_reset}"
 fi
 
+echo "Formatting frontend and markup files..."
+bazel run --ui_event_filters=-info,-stdout,-stderr --noshow_progress //tools/prettier:fix
+
 if ((gazelle)); then
-  echo "Running gazelle..."
-  bazel run //:gazelle
+  echo "Fixing BUILD dependencies with gazelle..."
+  bazel run --ui_event_filters=-info,-stdout,-stderr --noshow_progress //:gazelle
 fi
 
 git diff --name-only | sort >"$modified_after"
 
 echo 'All done!'
 
-# If no stdin is attached, skip interactive prompts.
-[ -t 0 ] || exit 0
+if ! ((stage)); then
+  exit
+fi
+if ! [ -t 0 ] || ! [ -t 1 ]; then
+  echo -e "${c_yellow}WARNING: not connected to a terminal; skipping interactive staging.${c_reset}"
+  exit 1
+fi
 
 export PATH="$PATH:$HOME/go/bin"
-if ! which fzf path-extractor &>/dev/null; then
-  echo -e "${c_green}TIP${c_reset}: Install fzf and path-extractor to interactively stage formatted files:"
-  echo '  go get -u github.com/edi9999/path-extractor/path-extractor'
-  echo '  go get -u github.com/junegunn/fzf'
-  exit
+if ! which fzf &>/dev/null; then
+  echo -e "${c_yellow}WARNING: Missing fzf tool; did not stage any changes.${c_reset}"
+  exit 1
 fi
 
 # Auto-commit only if there are no files staged before
```

Comment:
Change comment to drop the "Auto-", as "commit" appears to now be entirely manual?

Reply:
Done